packages/main/src/TableGroupRow.js

```diff
@@ -0,0 +1,144 @@
+import UI5Element from "@ui5/webcomponents-base/dist/UI5Element.js";
+import litRender from "@ui5/webcomponents-base/dist/renderer/LitRenderer.js";
+import { fetchI18nBundle, getI18nBundle } from "@ui5/webcomponents-base/dist/i18nBundle.js";
+import TableGroupRowTemplate from "./generated/templates/TableGroupRowTemplate.lit.js";
+import TableMode from "./types/TableMode.js";
+
+// Texts
+import {
+	TABLE_GROUP_ROW_ARIA_LABEL,
+} from "./generated/i18n/i18n-defaults.js";
+
+// Styles
+import styles from "./generated/themes/TableGroupRow.css.js";
+
+/**
+ * @public
+ */
+const metadata = {
+	tag: "ui5-table-group-row",
+	slots: /** @lends sap.ui.webcomponents.main.TableGroupRow.prototype */ {
+		/**
+		 * Defines the text of the component.
+		 * <br>
+		 * <b>Note:</b> Although this slot accepts HTML Elements, it is strongly recommended that you only use text in order to preserve the intended design.
+		 *
+		 * @type {Node[]}
+		 * @slot
+		 * @public
+		 */
+		"default": {
+			type: Node,
+		},
+	},
+	properties: /** @lends sap.ui.webcomponents.main.TableGroupRow.prototype */ {
+		/**
+		 * Defines the mode of the row (None, SingleSelect, MultiSelect).
+		 * @type {TableMode}
+		 * @defaultvalue "None"
+		 * @since 1.0.0-rc.15
```

Comment:
No since tag is needed here as the whole component is available since this version.

Reply:
Done